_release/bat/quotas_bat/quotas_bat_test.go

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package quotasbat is a placeholder package for the basic BAT tests.
+package quotasbat
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/01org/ciao/bat"
+)
+
+const standardTimeout = time.Second * 300
+
+func findQuota(qds []bat.QuotaDetails, name string) *bat.QuotaDetails {
+	for i := range qds {
+		if qds[i].Name == name {
+			return &qds[i]
+		}
+	}
+	return nil
+}
+
+func restoreQuotas(ctx context.Context, tenantID string, origQuotas []bat.QuotaDetails, currentQuotas []bat.QuotaDetails) error {
+	for i := range currentQuotas {
+		qd := findQuota(origQuotas, currentQuotas[i].Name)
```

Comment:
Do we need to compare qd against nil or are we guaranteed that findQuota will succeed?